### ISSUE TITLE
Default project setting value for `rendering/limits/global_shader_variables/buffer_size` exceeds maximum on Safari

### ISSUE THREAD

**Issue by adamscott** [godotengine/godot#91836]
### Tested versions

- Reproducible in 4.2.2.stable

### System information

Godot v4.3.dev (0712ce317) - macOS 14.4.1 - Vulkan (Forward+) - integrated Apple M2 - Apple M2 (8 Threads)

### Issue description

Running a project on Safari shows this error:

![Capture d’écran, le 2024-05-11 à 10 43 03](https://github.com/godotengine/godot/assets/270928/28faa77b-f795-493f-b609-8a89407d34ae)

```
[Error] USER WARNING: Project setting "rendering/limits/global_shader_variables/buffer_size" exceeds maximum uniform buffer size of: 16384
	onPrintError (index.js:15928)
	put_char (index.js:1455)
	write (index.js:1401)
	write (index.js:3477)
	doWritev (index.js:7816)
	_fd_write (index.js:7830)
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[__stdio_write]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[__vfprintf_internal]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[vfprintf]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[StdLogger::logv(char const*, void*, bool)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[Logger::logf_error(char const*, ...)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[Logger::log_error(char const*, char const*, int, char const*, char const*, bool, Logger::ErrorType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[CompositeLogger::log_error(char const*, char const*, int, char const*, char const*, bool, Logger::ErrorType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[OS::print_error(char const*, char const*, int, char const*, char const*, bool, Logger::ErrorType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[_err_print_error(char const*, char const*, int, char const*, char const*, bool, ErrorHandlerType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[_err_print_error(char const*, char const*, int, String const&, bool, ErrorHandlerType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[GLES3::MaterialStorage::MaterialStorage()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[RasterizerGLES3::RasterizerGLES3()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[RasterizerGLES3::_create_current()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[RendererCompositor::create()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[RenderingServerDefault::RenderingServerDefault(bool)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[Main::setup2()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[Main::setup(char const*, int, char**, bool)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[godot_web_main(int, char**)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[main]
	(fonction anonyme) (index.js:663)
	callMain (index.js:15378)
	(fonction anonyme) (index.js:16257)
	Promise
	(fonction anonyme) (index.js:16252)
[Error]    at: MaterialStorage (drivers/gles3/storage/material_storage.cpp:1114)
	onPrintError (index.js:15928)
	put_char (index.js:1455)
	write (index.js:1401)
	write (index.js:3477)
	doWritev (index.js:7816)
	_fd_write (index.js:7830)
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[__stdio_write]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[__vfprintf_internal]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[vfprintf]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[StdLogger::logv(char const*, void*, bool)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[Logger::logf_error(char const*, ...)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[Logger::log_error(char const*, char const*, int, char const*, char const*, bool, Logger::ErrorType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[CompositeLogger::log_error(char const*, char const*, int, char const*, char const*, bool, Logger::ErrorType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[OS::print_error(char const*, char const*, int, char const*, char const*, bool, Logger::ErrorType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[_err_print_error(char const*, char const*, int, char const*, char const*, bool, ErrorHandlerType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[_err_print_error(char const*, char const*, int, String const&, bool, ErrorHandlerType)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[GLES3::MaterialStorage::MaterialStorage()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[RasterizerGLES3::RasterizerGLES3()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[RasterizerGLES3::_create_current()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[RendererCompositor::create()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[RenderingServerDefault::RenderingServerDefault(bool)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[Main::setup2()]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[Main::setup(char const*, int, char**, bool)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[godot_web_main(int, char**)]
	godot.web.template_release.dev.wasm32.nothreads.wasm.wasm-function[main]
	(fonction anonyme) (index.js:663)
	callMain (index.js:15378)
	(fonction anonyme) (index.js:16257)
	Promise
	(fonction anonyme) (index.js:16252)
```

Running that same project on Firefox doesn't error out, as the WebGL 2 Max Uniform Block Size is of 65536.

### Steps to reproduce

1. Create a project with default values
2. Export a Web build
3. Open the Web build with Safari
4. See the error.

If you set `rendering/limits/global_shader_variables/buffer_size.web` to 16384, it doesn't error out.

### Minimal reproduction project (MRP)

n/a

**Comment by clayjohn** [comment]
Duplicate of https://github.com/godotengine/godot/issues/85374

**Comment by adamscott** [comment]
Closed as duplicate of #85374.
